minigalaxy/launcher.py

```diff
@@ -130,10 +126,11 @@ def get_windows_exe_cmd(game, files):
                         if "category" in task and task["category"] == "game" and "path" in task:
                             working_dir = task["workingDir"] if "workingDir" in task else "."
                             path = task["path"]
-                            exe_cmd = [get_wine_path(game), "start", "/b", "/wait", "/d", working_dir,
-                                       path]
+                            exe_cmd = [get_wine_path(game), "start", "/b", "/wait",
+                                       "/d", f'c:\\game\\{working_dir}',
```

Comment:
I've had a look into some of the gog*info files used in the test. It appears as if the workingDir property as well as the executable path always are relative paths.
The documentation for start is pretty sparse.
start runs within wine. It's normally expecting windows type paths. If you don't specify an absolute path, it will resolve with the current working directory.
Anything coming from linux is normally translated to z:\, e.g. z:\home\user\GOG Games\somegame. While it may be the same directory in the linux workspace, for wine they are different.
That's why i made it absolute here. If you happen to know or own a game that requires a different working directory, please let me know and i see if can also test and verify that.